src/regex_manager.h

```diff
@@ -27,17 +28,20 @@ class RegexManager : public BanjaxFilter
     std::unique_ptr<RE2> re2_regex;
     unsigned int interval; //interval to look in mseconds
     float rate; //threshold /interval
+    std::unordered_set<std::string> hosts_to_skip;
```

Comment:
`std::unordered_set` uses some hash function so lookups are constant-time. Most often, the set will be empty or have ~1 member, though, so I guess the constant factors in the algorithmic complexity will dominate. Again, I'm never sure how pedantic I'm supposed to be in C++ -- I guess the only way to really pick the best data structure would be to benchmark it, but that feels like I'm going down a rabbit hole...

Reply:
i'm leaning towards using a naiive
```
for string in strings:
   if strcomp(string, host)
```
(whatever that's called in `std`) instead of the fancy hashing set.

Reply:
I think std::unordered_set will do the job here. Empty or not, it will be O(1).